docs/static/releasenotes.asciidoc

```diff
@@ -65,6 +66,75 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-6]]
+=== Logstash 7.17.6 Release Notes
+
+[[notable-7-17-6]]
+==== Notable issues fixed
+
+* Support Ironbank Docker build https://github.com/elastic/logstash/pull/14298[#14298]
+* Fix artifact path for integration with ubireleaser https://github.com/elastic/logstash/pull/14319[#14319]
+
+[[dependencies-7-17-6]]
+==== Updates to dependencies
+
+* Pin bundler 2.3.18 to avoid bug in .19 https://github.com/elastic/logstash/pull/14394[#14394]
```

Comment:
I don't think we need this

Reply:
To be clear before I delete the wrong thing... By "this" you meanLine 81 about Bundler?

Reply:
Yes, I don't think it is something that users need to know